pkg/pool-utils/contracts/BasePool.sol

```diff
@@ -36,6 +36,10 @@ import "./BasePoolAuthorization.sol";
  * @dev Reference implementation for the base layer of a Pool contract that manages a single Pool with optional
  * Asset Managers, an admin-controlled swap fee percentage, and an emergency pause mechanism.
  *
+ * This Pool pays protocol fees by minting BPT directly to the ProtocolFeeCollector instead of using the
+ * `dueProtocolFees` return value. This results in better rates for users, as the Pool won't unbalance itself right
+ * before they join/exit. Furthermore, the underlying tokens will continue to provide liquidity for traders.
```

Comment:
> This results in better rates for users, as the Pool won't unbalance itself right before they join/exit. 
 
 This may be not clear if you do not know the previous implementation. "Why would it unbalance?" would be my question if I never read the LegacyBasePool.

Reply:
```suggestion
 * This Pool pays protocol fees by minting BPT directly to the ProtocolFeeCollector instead of using the
 * `dueProtocolFees` return value, which would cause the Vault to transfer pool tokens and unbalance the pool.
 * Paying in BPT results in better rates for users, as the Pool won't unbalance itself right before the join/exit.
 * Furthermore, the underlying tokens will continue to provide liquidity for traders.```

Reply:
Suggested change 
 * before they join/exit. Furthermore, the underlying tokens will continue to provide liquidity for traders.
 * This Pool pays protocol fees by minting BPT directly to the ProtocolFeeCollector instead of using the
 * `dueProtocolFees` return value, which would cause the Vault to transfer pool tokens and unbalance the pool.
 * Paying in BPT results in better rates for users, as the Pool won't unbalance itself right before the join/exit.
 * Furthermore, the underlying tokens will continue to provide liquidity for traders.```

Reply:
Spot on comment, thank you. I've updated the wording.